CHANGES.rst

```diff
@@ -380,6 +380,12 @@ astropy.tests
 astropy.time
 ^^^^^^^^^^^^
 
+- Change epoch of ``TimeUnixTAI`` (``"unix_tai"``) from ``1970-01-01T00:00:00 UTC``
```

Comment:
This should also be put into the API change section.

Reply:
Where would that go? We decided this is a bug fix which should be applied to 4.2.x, there is no API changes section for this, since it is a bugfix release.

Reply:
Or do you mean we should change the change log entry for astropy 4.2?

Reply:
Normally we don't make API changes in a bug fix release, but this is definitely an API change that needs to be highlighted. So that would mean either creating an API change section, or else deciding that we should not break the normal rules and instead just target 4.3.

Thoughts @mhvk or @pllim or @bsipocz ?

Reply:
I think this is really a bug fix, as we concluded that the code just gave the wrong answer before. It doesn't seem an API change - no "application program interface" was changed after all!

For highlighting this change, perhaps it should go up front? Or in "other changes"?

Though perhaps there is not that much difference from other bug fixes which have happened before without (I think) special mention, which also may have caused real problems:
```
- Fixed use of weights with ``LinearLSQFitter``. [#10687]

- Fix ``with_H0`` equivalency to use the correct direction of
  conversion. [#8292]
```

Reply:
p.s. In the end, a main lesson I am learning from this is to be thankful how well astropy documents things! The fact that it is so difficult to find a precise definition of `CLOCK_TAI` is concerning!

Reply:
Occasionally there were API changing bug fixes, mostly in lts and mostly for utils stuff that are needed downstream. Either add an API change section or other changes, for examples look for the 2.0.x sections. 
However I would be very cautious making a real API change. If it's a bug then it's not really an API change but a fix and can land in the appropriate section

Reply:
@mhvk - I think of an API change including the *specification* of a function that causes the code to give a different output for the same input.  It is certainly possible that someone has written code like below, so it would be good to highlight this change.
```
t_tai = Time(tm).unix_tai + 8
...
```
The code gave the correct answer given the specification, but the spec (code "requirements") was wrong. I tend to think of that as API, but maybe it's just me.

Reply:
The point is, the code before did not match the documented definition. So this is really a fix that brings the code in agreement with the docs.

Reply:
Indeed (to both). I think the main question really is how to ensure this gets seen. Easiest would seem to have a second entry in "Other changes".

Reply:
The "Known Issues" document seems to mainly handle higher level problems like numpy integration (and some things look lilke they could be removed because they belong to versions of astropy or dependencies that are not supported anymore).

I am not sure how best to proceed.